docs/src/build_tips.md

```diff
@@ -0,0 +1,135 @@
+# Building tips
+
+BinaryBuilder provides a convenient environment to enable cross-platform building. But, many libraries have complicated build scripts that may need to be adapted to support all of the BinaryBuilder targets.
+
+## Initiating different shell commands based on target
+
+Sometimes, you need to adapt build scripts based on the target platform. This can be done within the shell script. Here is an example:
+
+```sh
+if [[ ${target} == *-apple-* ]]; then
+    export CC=clang
+    export CXX=clang++
+    export LDFLAGS="-mmacosx-version-min=10.8"
+fi
+```
+
+Here are examples of scripts with target-specific checks:
+
+* [Keno/ReadStatBuilder](https://github.com/Keno/ReadStatBuilder/blob/master/build_tarballs.jl#L35-L39) - windows check
+* [JuliaDiffEq/SundialsBuilder](https://github.com/JuliaDiffEq/SundialsBuilder/blob/cmake-mod/build_tarballs.jl#L125-L131) - 32-bit check
+* [staticfloat/IpoptBuilder](https://github.com/staticfloat/IpoptBuilder/blob/master/build_tarballs.jl#L28-L32) - apple check
+
```

Comment:
Let's add OpenBLASBuilder to this list.  Also make sure to canonicalize your URLs (press `y` in your browser window) so that when the files change, the line numbers (`#L28-32`, etc...) don't point to random chunks of text.

Reply:
For ReadStatBuilder probably better to point to my repo, that is the one I will use for ReadStat.jl.

Reply:
Interesting. I didn't know about `y`. Makes sense!